src/main/java/Task.java

```diff
@@ -0,0 +1,20 @@
+abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getDescription() {
+        return description;
+    }
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+    public void setDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+    public abstract String toString();
+    public abstract String toFileString();
```

Comment:
Consider making then non abstract methods, as they can be easily utilized by sub-classes. As such, Task could not be a abstract class.